source/includes/azure/_instances.md

```diff
@@ -247,6 +247,11 @@ curl -X DELETE \
 
 Delete an existing instance.
 
+<aside class="notice">
```

Comment:
Shouldn't we also add the doc for the optional query param `deleteOsDisk`. It is possible to specify you want to delete the OS disk through the API right?

Reply:
As of now, that's not the case. The endpoint method is not taking any query param for deleting OS disk.

Reply:
Can you test it? I suspect that it will delete the os disk because we add any query params to the entity https://github.com/cloudops/cloudmc-core/blob/57ce2a9b5a47bf6896288e9917bd466ed63c4229/src/main/java/com/cloudops/mc/rest/resource/services/ServiceEntityResource.java#L759

Reply:
Yes you might be right as its taking query param out of `uriInfo` and adding it to the entity. But will test it to make sure.

Reply:
5ecbdbb